src/bun.js/node/path_watcher.zig

```diff
@@ -239,16 +239,8 @@ pub const PathWatcherManager = struct {
                                     if (path.len <= 1) {
                                         continue;
                                     }
-                                    if (path.len == 0) {
-                                        while (path.len > 0) {
-                                            if (bun.strings.startsWithChar(path, '/')) {
-                                                path = path[1..];
-                                                break;
-                                            } else {
-                                                path = path[1..];
-                                            }
-                                        }
-                                    } else {
+
+                                    if (bun.strings.startsWithChar(path, '/')) {
```

Comment:
Nice find. Should we skip more than one with `strings.withoutLeadingSlash()`?

Reply:
I `don't think is possible at this point to get more than one `/` I did some tests and we should be good with this.

Reply:
👍